src/env-dev.json

```diff
@@ -5,7 +5,7 @@
     "ws": "ws://127.0.0.1:8546"
   },
   "network-id": 4,
-  "auto-login": true,
+  "auto-login": false,
```

Comment:
Is this setting actually doing anything right now?

Reply:
I didn't track down how it's used. I've set it to false so I can use metamask and not local unlocked node. When searching the codebase I didn't find any usages.
Apparently it's superstitious behavior. Still recommend setting to false, we need not to accidently use it. 😃